diff --git a/options_win.cmake b/options_win.cmake
index 58d8941..e60f97d 100644
--- a/options_win.cmake
+++ b/options_win.cmake
@@ -91,18 +91,20 @@ if (MSVC)
             /LARGEADDRESSAWARE # Allow more than 2 GB in 32 bit application.
         )
     endif()
-
     if (DESKTOP_APP_SPECIAL_TARGET)
-        target_compile_options(common_options
-        INTERFACE
-            /WX
-            $<IF:$<CONFIG:Debug>,,/GL>
-        )
-        target_link_options(common_options
-        INTERFACE
-            $<IF:$<CONFIG:Debug>,,/LTCG>
-            $<IF:$<CONFIG:Debug>,,/LTCGOUT:>
-        )
+        # don't do /LTCG sa free GH runner is out of heap on x64 build
+        # /GL implicitly enables /LTCG
+        # /LTCGOUT: - does not make sense w/o /LTCG
+        #target_compile_options(common_options
+        #INTERFACE
+        #    /WX
+        #    $<IF:$<CONFIG:Debug>,,/GL>
+        #)
+        #target_link_options(common_options
+        #INTERFACE
+        #    $<IF:$<CONFIG:Debug>,,/LTCG>
+        #    $<IF:$<CONFIG:Debug>,,/LTCGOUT:>
+        #)
     endif()
 else()
     target_compile_definitions(common_options
