diff --git a/options_win.cmake b/options_win.cmake
index 8a8265c..33a4ba5 100644
--- a/options_win.cmake
+++ b/options_win.cmake
@@ -96,16 +96,19 @@ if (MSVC)
     endif()
 
     if (DESKTOP_APP_SPECIAL_TARGET)
-        target_compile_options(common_options
-        INTERFACE
-            /WX
-            $<$<NOT:$<CONFIG:Debug>>:/GL>
-        )
-        target_link_options(common_options
-        INTERFACE
-            $<$<NOT:$<CONFIG:Debug>>:/LTCG>
-            $<$<NOT:$<CONFIG:Debug>>:/LTCGOUT:>
-        )
+        # don't do /LTCG sa free GH runner is out of heap on x64 build
+        # /GL implicitly enables /LTCG
+        # /LTCGOUT: - does not make sense w/o /LTCG
+        #target_compile_options(common_options
+        #INTERFACE
+        #    /WX
+        #    $<$<NOT:$<CONFIG:Debug>>:/GL>
+        #)
+        #target_link_options(common_options
+        #INTERFACE
+        #    $<$<NOT:$<CONFIG:Debug>>:/LTCG>
+        #    $<$<NOT:$<CONFIG:Debug>>:/LTCGOUT:>
+        #)
     endif()
 endif()
 
